### ISSUE TITLE
Contribution clarification

### ISSUE THREAD

**Issue by JesperKawuma** [Mboalab/Outreachy_December_2022-March_2023-Internships-Typhoid-diagnostics-Project#18]
Hello everyone, I am Jesper Kawuma an Outreachy applicant from Uganda. I need some help with the contribution. Can someone please clarify a bit on what's supposed to be done?

**Comment by ruqaiyasattar** [comment]
Hi, I am an outreachy applicant too! All you have to do is to fork this repo, create a branch,  make changes in the uploaded file in the same branch which you created, commit and send PR! 
this resource will be helpful:    https://skills.github.com/?fbclid=IwAR0N6aF4fnGdYJEpLy7e7quWKVAd-wCkxvUTLZScWvPJMt5Iwr3kYunfJ_s ! 

Please read the contribution Instructions for more clear understanding of the task! 

 https://github.com/Mboalab/Outreachy_December_2022-March_2023-Internships-Typhoid-diagnostics-Project/blob/main/Project%20Contribution%20Information/Contributions.md

I hope this helps!

**Comment by DellahM** [comment]
Thank you @ruqaiyasattar for the explanation.

**Comment by Greatness01** [comment]
>Thank you @ruqaiyasattar  for the explanation, does this apply for task0 too?

**Comment by PetraAG** [comment]
yes. also join the community slack and whatsapp groups from the outreachy platform for better collaboration and interaction.

**Comment by Greatness01** [comment]
Okay

Thanks
